hack/update-go-module-dependencies.sh

```diff
@@ -1,4 +1,4 @@
-#!/bin/bash -ex
+#!/bin/bash -e
```

Comment:
Since the -x to print the commands is being removed, would it make sense to add another echo in line 67 where openshift modules are being replaced?

Reply:
Added better logging to the script.